cmd/server/main.go

```diff
@@ -1,9 +1,53 @@
 package main
 
-import "github.com/multipleton/sa-4/engine"
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/multipleton/sa-4/command"
+	"github.com/multipleton/sa-4/engine"
+)
 
 func main() {
+	inputFile := os.Args[1]
+	if _, err := os.Stat(inputFile); errors.Is(err, os.ErrNotExist) {
+		fmt.Println("invalid file name.")
+		return
+	}
 	eventLoop := new(engine.EventLoop)
 	eventLoop.Start()
-	// TODO
+	if input, err := os.Open(inputFile); err == nil {
+		defer input.Close()
+		scanner := bufio.NewScanner(input)
+		for scanner.Scan() {
+			commandLine := scanner.Text()
+			cmd := parse(commandLine)
+			eventLoop.Post(cmd)
+		}
+	}
+	eventLoop.AwaitFinish()
+}
+
+func parse(line string) engine.Command {
+	arr := strings.Fields(line)
+	if arr[0] == "print" {
+		if len(arr) == 2 {
+			return &command.Print{Value: arr[1]}
+		}
+		return &command.Print{Value: "SYNTAX ERROR: invalid arguments count"}
+	}
+	if arr[0] == "split" {
+		if len(arr) == 3 {
+			if len(arr[2]) == 1 {
+				r := []rune(arr[2])
+				return &command.Split{Input: arr[1], Separator: r[0]}
+			}
+			return &command.Print{Value: "SYNTAX ERROR: separator should be a single character"}
+		}
+		return &command.Print{Value: "SYNTAX ERROR: invalid arguments count"}
+	}
+	return &command.Print{Value: "SYNTAX ERROR: invalid command"}
```

Comment:
Errors during interpretation should not be putted in the result file. They should be printed to the terminal.

From issue:
> _Should parse commands from file and add them to MessageQueue or **in case of error - throw exception**_

Reply:
![image](https://user-images.githubusercontent.com/52866008/147414972-405eae0d-2a70-4c4a-8596-e9c30a21b967.png)

Reply:
👍